keycloak_add_users.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
```

Comment:
I'm not sure about wanting this kind of mechanism to create users for the tests, as it adds some non-related-to-mbop tests logic that we'd have to maintain.

We have a realm export file that imports users added there in the corresponding node, that just requires adding users as XML definitions. It is ugly, but it just requires to add more users as configuration. 

While I agree it'd be great to have a better way to create user fixtures, I believe that implementing a system that interacts with Keycloak to do it is something far more involved than what we need. I'm happy to discuss how to create users for our tests if you think the current solution doesn't fit the needs, but I think for now doing something like [what I'm trying to do here](https://github.com/RedHatInsights/mbop/commit/42a8be3b1fbfd60d37ecf151cdf04ae674920bf2) solves the problem of adding users for tests without adding logic to create users to the test collection. 

Please let me know your thoughts.